docs/data/material/components/pickers/deprecated-pickers.md

```diff
@@ -0,0 +1,19 @@
+# Date / Time pickers
```

Comment:
Is there a reason for capitalizing Time here? The headline seems to be referring to the general category of these components, as opposed to `<DatePicker>` and `<TimePicker>` themselves. 

Also, I think it looks a lot cleaner to ditch the "/" in favor of "and."

```suggestion
# Date and time pickers
```

Reply:
I'd suppose it's because you can also have `<DateTimePicker>`. But I personally think it's way cleaner the way you're proposing here.